content/docs/guides/continuous-delivery/octopus-deploy.md

```diff
@@ -0,0 +1,118 @@
+
+## Sample Project
+
+For the sake of this walkthrough, we will try to deploy the simple AWS example located [here](https://github.com/pulumi/examples/tree/master/aws-ts-hello-fargate). The example deploys a containerized Python Flask app on [AWS Fargate](https://aws.amazon.com/fargate/). This example also shows a pattern that is common with Pulumi -- keeping your infrastructure app (Pulumi) with your actual application code, but certainly not necessary. Regardless of how you decide to structure your project, you will need to provide the package to Octopus somehow.
+
+Alternatively, you can also run `pulumi new [template]` to create a template project.
+Learn more [here]({{< ref "/docs/reference/cli/pulumi_new.md" >}}).
+
+## Stack and Branch Mappings
+
+The scripts below act on a hypothetical stack: `homer/acme-org/python-app`.
+You can create a new stack by running [`pulumi stack init`]({{< relref "/docs/reference/cli/pulumi_stack_init.md" >}}) if you have already created a project.
+
+**Note**: The names used above are purely for demonstration purposes only.
+You may choose a naming convention that best suits your organization.
+
+## PULUMI_ACCESS_TOKEN
+
+To login non-interactively in to the CLI, you will need to set the env var `PULUMI_ACCESS_TOKEN` as a [project variable](https://octopus.com/docs/deployment-process/variables). To create a new access token, go the [Access Tokens](https://app.pulumi.com/account/tokens) page on the Pulumi Console.
+
+## Octopus Setup
+
+### Infrastructure
+
+Infrastructure in Octopus is represented as deployment targets and workers (tentacles or SSH machines). Additionally, you can also have environments which is a logical grouping of deployment targets and workers that represents each of your environments.
+
+In a typical scenario where Pulumi is creating your cloud infrastructure, you will only need to a worker that can run the Pulumi CLI commands against your code package.
+
+### Project
+
+A project represents the application being deployed. A project uses versioned packages (which you will read about in the following section) and uses variables during the deployment process.
+
+### Packages
+
+Packages can be your source code bundled-up in one of the [supported formats](https://octopus.com/docs/packaging-applications#supported-formats). They can also just be a handlful of scripts that you may want to use during your deployment process.
+
+In order to create a package, Octopus offers several ways that you can integrate into your existing build (CI) system. [Learn more](https://octopus.com/docs/packaging-applications/create-packages) about the options available to you for packaging your apps.
+
+For Pulumi apps, you can simply package the entire Pulumi app and extract the bundled package onto a worker where the Pulumi CLI can access them.
+
+## Create and configure a project
+
+A [deployment process](https://octopus.com/docs/deployment-process) is comprised of the following steps:
```

Comment:
Nit: `An [Octopus deployment process](...)` might be clearer, IMHO.

Reply:
I added more clarification to these sections based on your previous [comment](https://github.com/pulumi/docs/pull/1823#discussion_r332128032).